docs/source/athena-user-guide.rst

```diff
@@ -0,0 +1,59 @@
+Athena User Guide
+=================
+
+Concepts
+--------
+
+* `SQL <https://www.w3schools.com/sql/sql_intro.asp>`_
```

Comment:
It's not clear if Presto commands are actually supported, AWS just mentions normal SQL in documentation